pkg/boot/universalpayload/utilities.go

```diff
@@ -41,10 +41,10 @@ const (
 const (
 	rsdpTableOffset  = 0x1000
 	tmpHobOffset     = 0x2000
-	tmpStackOffset   = 0x3000
-	tmpStackTop      = 0x4000
-	trampolineOffset = 0x4000
-	uplImageOffset   = 0x5000
+	tmpStackOffset   = 0x4000
+	tmpStackTop      = 0x5000
+	trampolineOffset = 0x5000
+	uplImageOffset   = 0x6000
```

Comment:
Is there a limit on maximum offset we can use ?

Reply:
Good question, @AjanZhong do you know how much space we're working with?

Reply:
> Good question, @AjanZhong do you know how much space we're working with?

We will do relocation for UPL FIT image based on the uplImageOffset, which means, we can feel free to increase Offset size.

Reply:
great, it sounds like this is position independent

Reply:
> > Good question, @AjanZhong do you know how much space we're working with?
> 
> We will do relocation for UPL FIT image based on the uplImageOffset, which means, we can feel free to increase Offset size.

@AjanZhong is there any theoretical limit ?  -- e.g. we can't bypass 4GiB limit in 32bit OR you are saying this is truly limitless

Reply:
About theoretical limit, the answer is yes. uplImageOffset equals to the size of required stuffs including: (DTB, ACPI data, Hob, Stack, trampoline code).
In current design, we need to find an available contiguous physical address space to place required stuffs with size of uplImageOffset, and place FIT image with size about 5MB.
So if we set this offset too large (for instance more than 1GB), we might fail to find available contiguous physical address space in real hardware, especially in embedded environment.